### DISCUSSION TITLE
Change Command Failed EVR to Command Severity

### DISCUSSION THREAD

**Discussion by LeStarch** [nasa/fprime#1095] [Call for Comments]
Currently the Command Dispatcher uses `WARNING_HI` severity when reporting a command has failed to execute.  The proposal is to use the `COMMAND` severity instead.  This is for several reasons:

1. Most users like to see `COMMAND severity ... ... ... COMMAND severity` blocks in the EVR log to know what time/events the command bracketed.  This currently only works when commands succeed.
2. `WARNING_HI` EVRs typically are used for indicating a cross-component problem.  However, a command failure isn't always a problem.  Criticality of a command failure is project or component dependent.
3. `COMMAND` severity EVRs get special GUI processing to replace OpCodes for ease-of-use.  This is not applied to `WARNING_HI` making command failures difficult to diagnose.
4. Most components will report their own more-detailed `WARNING_HI` EVRs to indicate why a command failed.  Having an extra `WARNING_HI` saying the command failed isn't adding much.

@metzdigital could you elaborate if I missed anything?

**Comment by metzdigital** [comment]
You've captured the reasons perfectly, thanks @LeStarch!

**Comment by LeStarch** [comment]
Notes:
- NO_OP testing components should emit appropriate WARNINGs
- Should we emit 2 EVRs WARNING_HI and Command complete: add to checklist, not force warning
- Add to checklist.
- Ensure this only affects properly dispatched commands

**Comment by LeStarch** [comment]
This one has been approved.
